packages/suite-desktop-api/src/api.ts

```diff
@@ -90,7 +90,7 @@ export interface DesktopApi {
     getHttpReceiverAddress: DesktopApiInvoke<'server/request-address'>;
     // Tor
     getTorStatus: DesktopApiSend<'tor/get-status'>;
-    toggleTor: DesktopApiSend<'tor/toggle'>;
+    toggleTor: DesktopApiInvoke<'tor/toggle'>;
```

Comment:
CI validation fails because you've changed toggleTor message from send to invoke, but you didnt update unit tests in desktop-api package.
I would separate all changes regarding desktop-api package (including modified tests) and suite-desktop implementation. [reference example](https://github.com/trezor/trezor-suite/pull/5632/commits)